### PR TITLE
Remove space in output_prompt_overwrite

### PR DESCRIPTION
**Pull request by jakobnissen**
Fixes issue #211 
This will change the prompt behaviour of users who overwrite `OUTPUT_PROMPT`, but is it trivial to restore the old behaviour.

### CODE REVIEW

**Comment by KristofferC** [comment]
There is some 1.4 vs 1.5 difference here:

``` 
❯ julia14 -q # 1.4
julia> using OhMyREPL

julia> 1
1

❯ julia -q # 1.5
julia> using OhMyREPL

julia> 1
 1 # <------------
```

**Comment by jakobnissen** [comment]
Are you sure it's not just because your 1.4 points to a different version of OhMyREPL? I can't reproduce (also, it would be strange if this was something with a Julia version):

```
OhMyREPL.jl on  hexint [?] via C base took 35s ❯ /Applications/Julia-1.4.app/Contents/Resources/julia/bin/julia -q
julia> using OhMyREPL

julia> 1
 1

julia> exit()

OhMyREPL.jl on  hexint [?] via C base took 2s ❯ julia -q
julia> using OhMyREPL

julia> 1
 1
```

**Comment by KristofferC** [comment]
Yeah, that was it. However, I think we should just remove the space by default? Right now you get the weird space just by loading OhMyREPL. Also, for consistency, should probably remove the space at

https://github.com/KristofferC/OhMyREPL.jl/blob/dc09f6508ce2efaad1a90d072ac42782953c9fe0/src/prompt.jl#L12

as well?

File: ohmyrepl_precompile.jl [hunk 1]
```diff
@@ -0,0 +1,148 @@
+precompile(Tuple{typeof(Base.CoreLogging.with_logger), Function, Logging.ConsoleLogger})
```
Comment (KristofferC) [inline ohmyrepl_precompile.jl:1]: accident?

**Comment by jakobnissen** [comment]
Oops, I completely borked up that commit. I also added the hex stuff. Let me fix it.

File: ohmyrepl_precompile.jl (hunk 1 above)
Comment (jakobnissen) [reply ohmyrepl_precompile.jl]: Yes.

**Pull request merged**